fastlane/Fastfile

```diff
@@ -2,24 +2,24 @@ default_platform(:ios)
 
 platform :ios do
 
-  lane :test_iOS14_4 do
+  lane :test_iOS16 do
```

Comment:
We need to update the Continuous Integration Testing of the README with this new version.

Reply:
https://github.com/ably/ably-cocoa/pull/1543/commits/2812a5e55478bb324d280c9b9db49dfd09f6ba59